### PR TITLE
[21812] Add weekends to project cheetah calendar

### PR DESCRIPTION
**Pull request by LanceGundersen**
## Description
As a Veteran, I would like to be able to schedule a vaccine appointment on a weekend if there's an open slot.

## Testing done
Local, unit, and e2e

## Screenshots
Large Screen Single Slot On Tues/Thurs
image
Small Desktop Screen Single Slot On Tues/Thurs
image
Medium Screen Single Slot On Tues/Thurs
image
image
Smallest Screen Single Slot On Tues/Thurs
image

## Acceptance criteria
- [x] Changes are behind the va_online_scheduling_cheetah feature flag
- [x] Show all days of the week (Sun-Sat) by default for all months
- [x] Applies to direct book calendar only
- [x] Request calendar does not show weekends
- [x] DESIGN - All desktop, tablet, and mobile designs match the specs (copy is FPO)

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/project-cheetah/components/SelectDate1Page.jsx [hunk 1]
```diff
@@ -159,19 +159,17 @@ export function SelectDate1Page({
             {timezone &&
               ` Appointment times are displayed in ${timezoneDescription}.`}
           </p>
-          <p>
```
Comment (LanceGundersen) [inline src/applications/vaos/project-cheetah/components/SelectDate1Page.jsx:162]: I moved the `<ul></ul>` outside the `<p></p>` as it was causing a semantic error.

**Comment by jbalboni** [comment]
@LanceGundersen I think it needs some adjustment at smaller screen sizes:

Screen Shot 2021-03-29 at 10 27 56 AM

**Comment by LanceGundersen** [comment]
> @LanceGundersen I think it needs some adjustment at smaller screen sizes:
> 
> Screen Shot 2021-03-29 at 10 27 56 AM

Should be good to go. Note the transitions of the headers from abbreviation to fully written out.

**Comment by jbalboni** [comment]
@LanceGundersen Looking through it again, found some more issues to look at:

It doesn't look like the mobile view is matching the adjusted margins or button height that's in Peter's Sketch file. It looks to me like the margin between rows is a little bigger than the left/right margin between days. And on smaller screen sizes it looks like the days are not square, they're a little taller than they are wide, to add additional click area.

At smaller screen sizes, the checkmark overlaps the number:
Screen Shot 2021-03-30 at 11 45 57 AM

The positioning of slots underneath a date is a little wonky. I believe this is because there's code that aligns those slots left, right, or center depending on what position in the week the day is at, which was based on a 5 day week row.
Screen Shot 2021-03-30 at 11 45 44 AM
Screen Shot 2021-03-30 at 11 45 40 AM

**Comment by LanceGundersen** [comment]
> @LanceGundersen Looking through it again, found some more issues to look at:
> 
> It doesn't look like the mobile view is matching the adjusted margins or button height that's in Peter's Sketch file. It looks to me like the margin between rows is a little bigger than the left/right margin between days. And on smaller screen sizes it looks like the days are not square, they're a little taller than they are wide, to add additional click area.
> 
> At smaller screen sizes, the checkmark overlaps the number:
> Screen Shot 2021-03-30 at 11 45 57 AM
> 
> The positioning of slots underneath a date is a little wonky. I believe this is because there's code that aligns those slots left, right, or center depending on what position in the week the day is at, which was based on a 5 day week row.
> Screen Shot 2021-03-30 at 11 45 44 AM
> Screen Shot 2021-03-30 at 11 45 40 AM

@jbalboni I have some feedback from Peter on having the buttons be taller in the smaller views too for a good click target size of 44x44px to include the margins.

**Review by jbalboni — APPROVED** [review]
LGTM, I would merge in the latest from master to see if that unit test failure goes away.

**Pull request merged**